CHANGELOG.md

```diff
@@ -1,9 +1,10 @@
 # Change Log
 
-## [v0.10.0rc1](https://github.com/SeldonIO/alibi-detect/tree/v0.10.0rc1) (2022-06-01)
-[Full Changelog](https://github.com/SeldonIO/alibi-detect/compare/v0.9.1...v0.10.0rc1)
+## [v0.10.0](https://github.com/SeldonIO/alibi-detect/tree/v0.10.0) (2022-06-01)
+[Full Changelog](https://github.com/SeldonIO/alibi-detect/compare/v0.10.0...v0.9.1)
 
 ### Added
+- **New feature** TensorFlow is now an optional dependency. Error messages for incorrect use of detectors that are dependent on missing optional dependencies have been improved to include installation instructions and be more informative.
```

Comment:
Nitpick: not sure I would put this under Added and especially New Feature as that should relate only to new library functionality. I would personally put this under Changed, e.g. see https://github.com/SeldonIO/alibi/releases. Understand if opinions differ though.

Reply:
No, I think I agree, I've moved it to `Changed`